tests/testing_common.py

```diff
@@ -228,6 +232,59 @@ def get_grid_parameters(self, grid_parameters, filter_params_func=None):
         return generated_tests
 
 
+@contextmanager
```

Comment:
I'm wondering if the cm should be moved to `testing_utils.py` in case we want to apply it elsewhere (i.e. outside of  `PeftCommonTester`).

Reply:
Yep. I'd do this once we apply this to other tests.